src/commands/karma.js

```diff
@@ -209,6 +220,20 @@ class KarmaCommand extends Aquarius.Command {
 
     return;
   }
+
+  getEffectiveKarma(record) {
```

Comment:
Yes, the DB value of the karma count is unchanged until it gets incremented/decremented by a user.